src/test/java/org/example/XhtmlHelpWriter.java

```diff
@@ -0,0 +1,94 @@
+/*
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+ */
+package org.example;
+
+import static java.lang.String.format;
+
+import java.io.IOException;
+import java.util.Collection;
+import java.util.List;
+
+import org.apache.commons.cli.Util;
+import org.apache.commons.cli.help.AbstractHelpWriter;
+import org.apache.commons.cli.help.TableDef;
+
+/** An example XML helpWriter -- DO NOT USE as this does not properly escape strings */
```

Comment:
Hi @Claudenw 
I'm not crazy about providing a partial implementation. I think we either do it as a real example, and provide it as a nice show-and-tell in the docs, or not at all. Doing it right could validate the framework further (do we need a plugable escape function for example). The escaping can be done through Apache Commons Text's `org.apache.commons.text.StringEscapeUtils` for example. Depending on another Commons jar is OK since this is an example. An improved example could be done in second PR to make this PR easier to digest.

Reply:
OK.